tools/cmd/image.go

```diff
@@ -44,7 +44,7 @@ func imageSave(images []string, dest, clusterName string) error {
 		return fmt.Errorf("ERROR: couldn't save image stream to tarfile\n%+v", err)
 	}
 
-	log.Printf("INFO: saved images %s to [%s]", images, tarFileName)
+	l.Log().Printf("INFO: saved images %s to [%s]", images, tarFileName)
```

Comment:
This is still using logrus directly (and is not inside the k3d executable)
```suggestion
	log.Printf("INFO: saved images %s to [%s]", images, tarFileName)
```